.github/ISSUE_TEMPLATE.md

```diff
@@ -35,3 +35,14 @@ the backtick lines:
 ## What did you expect to happen instead of what actually happened?
 
 > Example: I expected a cinder volume snapshot to be successfully created.
+
+## Do you think the documentation team wants to know about this?
+
+[ ] Yes, and I submitted an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues "Docs issues")
```

Comment:
Aaack, I forgot that the checkboxes need a dash up front >> `- [ ] I LIKE PANCAKES`